custom/icds_reports/sqldata/base_operationalization.py

```diff
@@ -35,3 +39,42 @@ def rows(self):
                     0
                 ]
             ]
+
+
+class BaseOperationalizationBeta(ICDSMixin):
+
+    title = 'c. Status of operationalization of AWCs'
+    slug = 'operationalization'
+
+    @property
+    def headers(self):
+        return DataTablesHeader(
+            DataTablesColumn('', sortable=False),
+            DataTablesColumn('Sanctioned', sortable=False),
+            DataTablesColumn('Functioning', sortable=False),
+            DataTablesColumn('Reporting', sortable=False)
+        )
+
+    @property
+    def rows(self):
+
+        if self.config['location_id']:
+            filters = get_location_filter(self.config['location_id'], self.config['domain'])
+
+            if filters.get('aggregation_level') > 1:
```

Comment:
where does this get set for the mpr?

Reply:
from the above function `get_location_filter`